doc/FAQ.rst

```diff
@@ -59,49 +65,131 @@ Conversely, what Panel adds on top of Bokeh is full bidirectional communication
+|Provides widgets and layouts          | Yes             | Yes                  | Yes             | Yes                | Yes                    | Yes                |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Supports fully interactive plots      | Yes             | Yes                  | Yes             | No?                | Yes                    | Yes?               |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Supports incremental plot updates     | Yes             | Yes                  | Yes             | No?                | Yes                    | Yes?               |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Fully usable in Jupyter               | Yes             | Yes                  | Yes, with       | No                 | No, only via iframe    | No                 |
+|                                      |                 |                      | jupyter_bokeh   |                    |                        |                    |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Supports static HTML export           | Yes             | Not without a special| Yes             | No?                | No                     | No?                |
+|(for reports, websites, docs)         |                 | embedding procedure  |                 |                    |                        |                    |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Supports Matplotlib plots             | Yes             | Yes                  | No              | Yes                | With a separate adapter| No                 |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Supports Bokeh plots                  | Yes             | Yes                  | Yes             | Yes                | With a separate adapter| No                 |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Supports Plotly plots                 | Yes             | Yes                  | No              | Yes                | Yes                    | No                 |
```

Comment:
Plotly plots can be used in Shiny: https://plot.ly/r/shiny-tutorial/

Reply:
Thanks for the fix!